test/models/snapshot_item_test.rb

```diff
@@ -31,9 +31,9 @@ def test_relationships
   def test_validations
     instance = @snapshot_item_klass.new
 
-    instance.valid?
+    assert instance.invalid?
 
-    [:item_id, :item_type, :snapshot_id].each do |attr|
+    %i[item_id item_type snapshot_id object].each do |attr|
```

Comment:
Can we just keep the original form

```suggestion
    [:item_id, :item_type, :snapshot_id, :object].each do |attr|
```